Auth0/CredentialsManager.swift

```diff
@@ -652,6 +657,11 @@ public struct CredentialsManager {
                     dispatchGroup.leave()
                     return callback(.success(credentials))
                 }
+
+                guard self.allowsAutoRefreshing else {
```

Comment:
I'm not sure if this is right. This will prevent refreshing, but that's not the only thing the code below does, it also tries to read the credentials from the store/keychain